src/continuous_agg.c

```diff
@@ -577,7 +577,15 @@ ts_continuous_agg_drop_hypertable_callback(int32 hypertable_id)
 
 		if (data->raw_hypertable_id == hypertable_id)
 		{
-			continuous_agg_init(&ca, data);
+			/* schema might not exists at this point */
+			Oid nspid = get_namespace_oid(NameStr(data->user_view_schema), true);
+
+			if (OidIsValid(nspid))
+				ca.relid = get_relname_relid(NameStr(data->user_view_name), nspid);
+			else
+				ca.relid = InvalidOid;
+			ca.data = *data;
```

Comment:
shouldn't these 2 lines be conditional?  i.e.
i.e. this is valid only if you cna find a view object.
```suggestion
if( OidIsValid(...) )
{
            ca.redlid = ...
			ca.data = *data;
			drop_continuous_agg(...)
}
```

Reply:
Yes, you right. Missed that. But it should not be used anyway I believe since the relation marked as not found.